iiasa/message-ix-models
# Clean up Materials migration branch

This separate branch includes commits to tidy and make MESSAGEix-Materials run on message-ix-models.





## How to review

1. Check the removed files and changed code pieces
2. Build-Solve-Report MESSAGEix-Materials



## PR checklist


~- [ ] Continuous integration checks all ✅~ Not possible due to missing tests
- ~- Add or expand tests; coverage checks both ✅~ See https://github.com/iiasa/message-ix-models/pull/189#issuecomment-2128709498; will be handled via #194
